portal/config/eproms/QuestionnaireBank.json

```diff
@@ -769,6 +769,58 @@
           "rank": 0
         }
       ]
+    },
+    {
+      "resourceType": "QuestionnaireBank",
+      "classification": "baseline",
+      "name": "empro_post_intervention_baseline",
+      "start": "{\"days\": 0}",
```

Comment:
How do you indicate what event these are anchored on? eg for this case, it's the creation of a trigger (vs ironman main study using consent date)

Reply:
it's beyond the scope of the QB definition, for now.  handled via `trigger_states`.